build.sh

```diff
@@ -73,9 +73,10 @@ function init_options() {
 
   # Internal variable default values
   OC_OPTS=""
-  MAVEN_OPTS=""
+  MAVEN_OPTS="--batch-mode"
```

Comment:
Can't we set batch-mode only when running in a CI ? Having it as default doesn't feel right to me (I like colors ;-)

Reply:
true.